templates/summary_manager/_form/motivation.html.twig

```diff
@@ -1,3 +1,4 @@
 <h2 class="b__nudge--bottom">Mes motivations</h2>
 
-{{ form_row(summary_form.motivation) }}
+{{ form_errors(summary_form.motivation) }}
+{{ form_widget(summary_form.motivation, {'attr': {'placeholder': 'Expliquez en quelques lignes vos motivations'}}) }}
```

Comment:
We could keep the same spirit with something like: "Mes motivations expliquées en quelques lignes..."?